l10n-ms/src/main/java/com/example/l10nadmin/controller/L10nController.java

```diff
@@ -23,14 +21,47 @@
 public class L10nController {
 
 	private final L10nService l10nService;
+	private final L10nRepository l10nRepository;
 
 	@GetMapping("locale/{locale}")
 	@ApiOperation("Retrieves the localization strings for the requested language")
-	@ApiResponses(value = { @ApiResponse(code = 200, message = "Successful"),
+	@ApiResponses(value = {@ApiResponse(code = 200, message = "Successful"),
 			@ApiResponse(code = 401, message = "Unauthorized"),
-			@ApiResponse(code = 500, message = "System Error") })
+			@ApiResponse(code = 500, message = "System Error")})
 	public ResponseEntity<Map<String, String>> getLocalization(@PathVariable String locale) {
 		LOG.info("Retrieve locale texts for {}", locale);
 		return ResponseEntity.ok(l10nService.getLocalization(locale));
 	}
+
+	@GetMapping("l10n")
+	@ApiOperation("Returns all localization entries")
+	@ApiResponses(value = {@ApiResponse(code = 200, message = "Successful"),
+			@ApiResponse(code = 401, message = "Unauthorized"),
+			@ApiResponse(code = 500, message = "System Error")})
+	public ResponseEntity<List<L10n>> getAllLocalization() {
+		LOG.info("Retrieve all values");
+		return ResponseEntity.ok(l10nRepository.findAll());
+	}
+
+	@PostMapping("l10n/entry")
+	@ApiOperation("Creates a new entry")
+	@ApiResponses(value = {@ApiResponse(code = 200, message = "Successful"),
+			@ApiResponse(code = 401, message = "Unauthorized"),
+			@ApiResponse(code = 500, message = "System Error")})
+	public ResponseEntity postNewEntry(@RequestBody requestForm form) {
```

Comment:
A creation POST endpoint should normally return the newly created entry, in order for the caller to know its ID. It is returned by the repository.save() method.

Besides, ResponseEntity without template parameter yields warning in IJ